### DISCUSSION TITLE
What information is shared to those who sponsor me?

### DISCUSSION THREAD

**Discussion by JDJGInc** [community/community#138078] [Other Feature Feedback, Questions, & Ideas]
I noticed on Kofi it wanted to share my information, so I deleted that.
Does stripe on github share any info to those who sponsor me?
I'm in the us and I don't want my personal information shared

**Comment by Topromm** [comment]
When you receive sponsorships via GitHub using Stripe, your personal information is generally protected. GitHub doesn’t share sensitive personal details like your address or full name with sponsors UNLESS you explicitly choose to disclose that information.

As the sponsored user, you’ll mostly see the sponsor’s GitHub username and the amount they’re contributing, but their personal financial details are kept private, and the same goes for your information on their side.
